### PR TITLE
add user id implementation

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

File: cl-forest.lisp [hunk 1]
```diff
@@ -29,11 +29,15 @@
...
 
 (defvar *api-key* nil)
+(defvar *user-id* nil)
 
-(defun check-api-key ()
-  (unless (stringp *api-key*)
-    (error "API key is not set. Set the variable CL-FOREST:*API-KEY* to ~
-            a string containing your key.")))
+(defun check-credentials ()
+  (progn (unless (stringp *api-key*)
```
Comment (stylewarning) [inline cl-forest.lisp]: no need for `progn` here

File: cl-forest.lisp [hunk 2]
```diff
@@ -61,7 +65,8 @@ Return two values:
        ;;
        ;;    https://github.com/rigetticomputing/pyquil/blob/master/pyquil/forest.py#L234
        :accept "application/octet-stream"
-       :additional-headers `(("X-API-KEY" . ,*api-key*))
+       :additional-headers `(("X-API-KEY" . ,*api-key*)
+			     ("X-USER-ID" . ,*user-id*))
```
Comment (stylewarning) [inline cl-forest.lisp]: here and elsewhere, make sure your editor doesn't use tabs.

in emacs, you can remove all tabs in the file by doing `C-x h M-x untabify RET`

**Review by stylewarning — CHANGES_REQUESTED** [review]

File: cl-forest.lisp (hunk 1 above)
Comment (vtomole) [reply cl-forest.lisp]: Done.

File: cl-forest.lisp (hunk 2 above)
Comment (vtomole) [reply cl-forest.lisp]: Done.

**Pull request closed without merging**